backend/supabase/group_invites/functions.sql

```diff
@@ -0,0 +1,14 @@
+create
+or replace function get_last_week_long_link (this_group_id text) returns text as $$
```

Comment:
This function could kind of be a view. You just filter the view by group id after. I might prefer that style for next time.

Reply:
I'm trying to avoid views where I don't think it's super useful because that's another thing we have to pay attention to managing permissions for. Right now this table is admin read only, and I'd have to keep track of that for any views generated off of it